srcpkgs/removed-packages/template

```diff
@@ -492,6 +492,7 @@ replaces="
  vte290<=0.36.5_5
  vte<=0.28.2_16
  w_scan<=20170107_1
+ webkit2gtk-common<=2.38.1_1
```

Comment:
Do we really need this here? On an update, the package will be replaced by webkit2gtk even without this.